collections/_article/godotsharp-packages-net8.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Godot C# packages move to .NET 8"
+excerpt: "On moving to the latest .NET LTS release, and what it means for users."
+categories: ["progress-report"]
+author: Raul Santos
+image: /storage/blog/covers/progress-update-csharp-3.webp
+date: 2024-11-26 17:00:00
+---
+
+A new version of .NET is released every year on November, alternating Long Term Support (LTS) and Standard Term Support (STS) releases. The quality of all releases is the same. The only difference is the length of  support. LTS releases are supported for 3 years, while STS releases are supported for 1 year and a half.
+
+With the release of Godot 4.0 we moved our C# packages from targeting Mono to .NET 6. This version of .NET was released on November 8, 2021 and ended support on November 12, 2024.
+
+With .NET 6 ending support, Godot C# packages in 4.4 will now target the next LTS release .NET 8. Thanks to [RedworkDE](https://github.com/RedworkDE) and [Paul Joannon](https://github.com/paulloz) for the great work in [GH-92131](https://github.com/godotengine/godot/pull/92131).
+
+<figure>
+	<picture>
+		<source srcset="/storage/blog/dotnet/release-schedule-dark.svg" media="(prefers-color-scheme: dark)">
+		<img class="lightbox-ignore" style="background-color: transparent;" src="/storage/blog/dotnet/release-schedule-light.svg">
+	</picture>
+	<figcaption><small>.NET release cadence from the <a href="https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core">.NET official support policy</a>.</small></figcaption>
+</figure>
+
+## What does this mean for users?
+
+This means your project now needs to target .NET 8 or newer, otherwise it will be incompatible with the new `GodotSharp` packages. But you can still use libraries that target older .NET versions.
```

Comment:
```suggestion
Starting with Godot 4.4, your project will need to target .NET 8 or newer, otherwise it will be incompatible with the new `GodotSharp` packages. You will still be able to use other libraries that target older .NET versions.
```